src/features/mcda/components/MCDALayerEditor/MCDALayerParameters/TransformationsChart/TransformationsChart.tsx

```diff
@@ -0,0 +1,140 @@
+import { useMemo } from 'react';
+import { LineChart, Line, CartesianGrid, XAxis, YAxis, Tooltip, Legend } from 'recharts';
+import { scaleSqrt, scaleSymlog } from 'd3-scale';
+import { generateHclGradientColors } from '~features/mcda/utils/generateHclGradientColors';
+import { inViewCalculations } from '~core/logical_layers/renderers/stylesConfigs/mcda/calculations';
+import s from './TransformationsChart.module.css';
+import type { TransformationFunction } from '~core/logical_layers/renderers/stylesConfigs/mcda/types';
+import type { AxisTransformationWithPoints } from '~utils/bivariate';
+
+const CHART_GREEN = 'rgb(50, 170, 100)';
+const CHART_RED = 'rgb(228, 26, 28)';
+const CHART_YELLOW = 'rgb(251,237,170)';
+const COLOR_ORIGINAL = '#CCC';
+
+type TransformationsChartProps = {
+  transformedData?: AxisTransformationWithPoints;
+  originalData?: AxisTransformationWithPoints;
+};
+
+function getScaleFunction(
+  transformationFunction: TransformationFunction | undefined,
+  points: number[] | undefined,
```

Comment:
Can be 
```
  transformationFunction?: TransformationFunction,
  points?: number[],
```

or you want it to be more explicit?

Reply:
Yeah, here I wanted to be more specific to avoid making it look like the parameters are optional. We can just not have the data at the moment (that is a pretty early debug prototype after all)